docs/modules.md

```diff
@@ -0,0 +1,105 @@
+[toucan-sdk](README.md) / Exports
```

Comment:
Should I review this entire file as well?

Reply:
nope